code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -513,12 +513,14 @@
 	volume = 20
 	amount_per_transfer_from_this = 5
 	isGlass = FALSE
+	var/random_sprite = TRUE
```

Comment:
doc

Reply:
> 
> 
> doc

Alright added the proper documentation for var/random_sprite = TRUE